lib/filterx/object-json-array.c

```diff
@@ -78,6 +78,17 @@ _marshal(FilterXObject *s, GString *repr, LogMessageValueType *t)
   return TRUE;
 }
 
+static gboolean
+_repr(FilterXObject *s, GString *repr)
+{
+  FilterXJsonArray *self = (FilterXJsonArray *) s;
+
+  const gchar *json_repr = json_object_to_json_string_ext(self->jso, JSON_C_TO_STRING_PLAIN);
+  g_string_append(repr, json_repr);
```

Comment:
I think we should be consistent in `repr` implementations about how we use the GString parameter.

Some implementations append the string (`_string_repr`), some of them set it (`_bytes_repr`). I think we should fix all of them and I vote for `set`.